.circleci/src/commands/@mobile-commands.yml

```diff
@@ -109,6 +109,23 @@ mobile-release-ios:
     - run:
         name: fastlane build and upload
         command: cd packages/mobile/ios && bundle exec fastlane build_and_upload bundle_id:<<parameters.bundle-id>>
+    - run:
+        name: slack announce
```

Comment:
I think this should be it's own command that we pass parameters to like:
```
- slack-announce-deploy
    platform: 'mobile ios'
    path: packages/mobile
```

Reply:
yeah I just didn't want to refactor the entire protocol side of things too and we should converge that. given the protocol side already looks like this, I kind of want to leave as is for this initial merge just to see if we want to tweak things?

Reply:
Sure! I'm planning on cleaning up that root config.yml file anyway, so I could refactor this then